src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
@Nadahar As far i understood the code, the sample frequency is only a duplicated information, so no need to put it in our parsing.

I guess you're right for the explaination, and i don't find it was a good way to do that, whatever was they purpose...

Though, if you really want an other sample frequency log information, i can implemented `SamplingRate/String` and all your wishes will be exhausted.

But we need to agree on that before i do.